src/components/player-info.js

```diff
@@ -7,29 +7,58 @@ import { AVATAR_TYPES } from "../assets/avatars/avatars";
  */
 AFRAME.registerComponent("player-info", {
   schema: {
-    displayName: { type: "string" },
     avatarSrc: { type: "string" },
     avatarType: { type: "string", default: AVATAR_TYPES.LEGACY }
   },
   init() {
+    this.displayName = null;
     this.applyProperties = this.applyProperties.bind(this);
+    this.updateDisplayName = this.updateDisplayName.bind(this);
+    this.applyDisplayName = this.applyDisplayName.bind(this);
+
+    this.isLocalPlayerInfo = this.el.id === "player-rig";
```

Comment:
perhaps this should be a schema property instead of looking at the id?

Reply:
Oh, actually, I can't do that, since player-info is still networked, so we can't allow others to override that prop.